packages/core/src/command/feature-flags/index.ts

```diff
@@ -0,0 +1,154 @@
+import { errorToString } from "@metriport/shared";
+import { Config } from "../../util/config";
+import { out } from "../../util/log";
+import { capture } from "../../util/notifications";
+import { getFeatureFlags } from "./ffs-on-dynamodb";
+import {
+  BooleanFeatureFlags,
+  CxFeatureFlagStatus,
+  StringValueFeatureFlags,
+  stringValueFFsSchema,
+} from "../../external/aws/app-config";
+
+const { log } = out(`FFs`);
+
+export async function getFeatureFlagValueCxValues<T extends keyof StringValueFeatureFlags>(
+  region: string,
+  ffTableName: string,
+  cxId: string,
+  featureFlagNames?: T[]
+): Promise<CxFeatureFlagStatus> {
+  const configContentValue = await getFeatureFlags(region, ffTableName);
+  const targetFeatureFlags =
+    featureFlagNames && featureFlagNames.length > 0
+      ? featureFlagNames
+      : stringValueFFsSchema.keyof().options;
+  let response: CxFeatureFlagStatus = {};
+  targetFeatureFlags.map(featureFlagName => {
+    const featureFlag = configContentValue[featureFlagName];
+    if (featureFlag) {
+      response = {
+        ...response,
+        [featureFlagName]: {
+          cxInFFValues: featureFlag.values.includes(cxId),
+          ffEnabled: featureFlag.enabled,
+        },
+      };
+    }
+  });
+  return response;
+}
+
+export async function getFeatureFlagValueStringArray<T extends keyof StringValueFeatureFlags>(
+  region: string,
+  ffTableName: string,
+  featureFlagName: T
+): Promise<StringValueFeatureFlags[T]> {
+  try {
+    const configContentValue = await getFeatureFlags(region, ffTableName);
+    return configContentValue[featureFlagName];
+  } catch (error) {
+    const msg = `Failed to get Feature Flag Value`;
+    const extra = { featureFlagName };
+    log(`${msg} - ${JSON.stringify(extra)} - ${errorToString(error)}`);
+    capture.error(msg, { extra: { ...extra, error } });
+    return { enabled: false, values: [] };
+  }
+}
+
+export async function getFeatureFlagValueBoolean<T extends keyof BooleanFeatureFlags>(
+  region: string,
+  ffTableName: string,
+  envName: string,
```

Comment:
unnecessary, right?

Reply:
yep, good catch!

🤔 why didn't eslint complained about it.